PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -54,7 +54,8 @@ defaultSigma=$(echo "sqrt(200)" | bc -l)
 
 #TSC:should --path or --study-folder be the flag displayed by the usage?
 opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects" "--path"
-opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddMandatory '--subject-long' 'Subject' 'subject ID' "subject label, may be different from Session"
```

Comment:
This should be optional, otherwise it will break old Batch scripts (and the current batch script, too).

Reply:
ok